docs/mine-hnt/validators/testnet/deployment-guide.mdx

```diff
@@ -405,6 +405,55 @@ After running this, you'll need to input your wallet passphrase to sign the tran
 
 And with that, you're done. Congratulations! You're running a Helium Network Validator.
 
+## Overriding Default Configuration
```

Comment:
If you're really just replicating this maybe refer to the main net overriding section?

Reply:
thanks, great suggestion